internal/document_test.go

```diff
@@ -74,3 +113,448 @@ func TestDocument_Debug(t *testing.T) {
 		assert.ErrorContains(t, err, "PCR0 not found")
 	})
 }
+
+func TestDocument_Verify(t *testing.T) {
```

Comment:
Simply tests happy paths. All of the error cases are tested in the `CheckX` functions